op-program/client/l2/engineapi/block_processor.go

```diff
@@ -64,6 +64,8 @@ func NewBlockProcessorFromPayloadAttributes(provider BlockDataProvider, parent c
 	return NewBlockProcessorFromHeader(provider, header)
 }
 
+var emptyRequestsHash = types.CalcRequestsHash([][]byte{})
```

Comment:
You can use [`types.EmptyRequestsHash`](https://github.com/ethereum-optimism/op-geth/blob/optimism/core/types/hashes.go#L44) instead.

Reply:
nice, updated: [47f227f](https://github.com/ethereum-optimism/optimism/pull/13976/commits/47f227fc54a410a3a9c8db12a913385895b2a515)